installer/conf/container.conf

```diff
@@ -64,3 +64,17 @@
 <filter oms.container.**>
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path %STATE_DIR_WS%/out_oms_docker*.buffer
```

Comment:
Where are you replacing this string? [omsadmin.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/omsadmin.sh#L820) currently only replaces this string in a hardcoded set of files (search page for `update_path`).